app/views/layouts/headers/_current_user.html.erb

```diff
@@ -0,0 +1,31 @@
+<header>
```

Comment:
@jasperfurniss, @coleerikson, @ds29 FYI: I think this PR will create quite a few conflicts with the `new-layout` branch.